File: app/src/main/java/com/bogdanorzea/popularmovies/PopularMovies.java
package com.bogdanorzea.popularmovies;

import android.app.Application;

import timber.log.Timber;

public class PopularMovies extends Application {

    @Override
    public void onCreate() {
        super.onCreate();

        if (BuildConfig.DEBUG) {
            Timber.plant(new Timber.DebugTree());
        }
    }
}

File: app/src/main/java/com/bogdanorzea/popularmovies/PopularMovies.kt
package com.bogdanorzea.popularmovies

import android.app.Application

import timber.log.Timber

class PopularMovies : Application() {

    override fun onCreate() {
        super.onCreate()

        if (BuildConfig.DEBUG) Timber.plant(Timber.DebugTree())
    }
}
